concepts/strings-package/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "The strings package is included in the standard library and includes great utility functions to work with strings.",
```

Comment:
Simplification:

The standard library strings package includes great utility functions to work with strings.